app/assets/stylesheets/tags.scss

```diff
@@ -0,0 +1,27 @@
+.tag-list {
+  margin-top: 2rem;
+}
+
+.tags {
+  margin-bottom: 0.8rem;
+}
+
+.tag {
+  @extend .chip;
+  background-color: #E6E18F;
+  color: #92977E;
```

Comment:
Color literals like `#92977E` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#92977E` should be written as `#92977e`